client/landing/gutenboarding/stores/onboard/actions.ts

```diff
@@ -99,10 +101,12 @@ export function* createSite(
 
 	const currentDomain = domain ?? freeDomainSuggestion;
 	const siteUrl = currentDomain?.domain_name || siteTitle || username;
+	const isPublicSite = planSlug === 'ecommerce-bundle';
```

Comment:
The store is importing plan slugs from here.

```typescript
import {
	PLAN_FREE,
	PLAN_PERSONAL,
	PLAN_PREMIUM,
	PLAN_BUSINESS,
	PLAN_ECOMMERCE,
} from '../../../../lib/plans/constants';
```

We should probably be doing the same here, and in other places where we hard code plan slugs.